src/App/ComponentsDocumentation/components/InputField/index.js

```diff
@@ -61,16 +69,23 @@ const DeveloperDocumentation = () => (
         <section>
             <h3 className="mt-3">Autocomplete</h3>
             <p>Add <CodeTags type="primary" code="autocomplete"/> attributes to input fields when it is possible to make a form easier for users to fill. </p>
+            <p>Please check out <Link to="https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete">this site</Link> for full autocomplete specs.</p>
             <ComponentPreview language="html" showCasePanel codeFigure>
-                <InputGroup type="text" label="Name" autoComplete="name" id="developer-documentation-example" />
+                <form>
+                    <fieldset>
+                        <InputGroup type="text" label="Name" id="developer-doc-given-name-example" className="mb-3" autoComplete="name"/>
+                        <InputGroup type="text" label="E-mail" id="developer-doc-given-name-example" className="mb-3" autoComplete="email"/>
+                        <InputGroup type="number" label="Phone number" id="developer-doc-given-name-example" autoComplete="phone"/>
```

Comment:
the 3 inputs have the same `id` ?
`id="developer-doc-given-name-example"`
- no duplicate `id` attribute on 1 page
- especially no duplicate id for input fields since it will fails their label<->input functionality (now clicking on any of the 3 labels movesfocus to the 1st input)